src/core/analyzer.ts

```diff
@@ -90,7 +90,7 @@ export class Analyzer {
   static createForDirectory(dirname: string): Analyzer {
     return new Analyzer({
       urlLoader: new FSUrlLoader(dirname),
-      urlResolver: new PackageUrlResolver({})
+      urlResolver: new PackageUrlResolver({packageDir: dirname})
```

Comment:
Since this method creates a PackageUrlResolver, should the method name indicate that? Like `forPackageDirectory` or `forPackage`?

Reply:
I'm cool with `forPackage`, but my goal with this method is for us to use the best url resolver that we can infer for the directory. We don't have a lot of url resolvers at the moment, but I'm roughly thinking that this would be a place where a plugin could contribute to. (In a follow up PR this method is also made async, giving us more leeway).

So I don't think we'll always be using a PackageUrlResolver here.

Reply:
Plan is now to try to move this out into polymer-project-config. I'd like to keep it here for this PR, as we use it in, like, all of our tests right now.